### PR TITLE
Notes added to Enquiry

### PR DESCRIPTION
**Pull request by sanjaytkbabu**
# Description
Notes added to Enquiry
[PADS-192](https://apps.nrs.gov.bc.ca/int/jira/browse/PADS-192)

## Types of changes

New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by slhurley** [comment]
We will also need to update the link for BF notes on the "My bring forward notifications" and the "BF Calendar" view.  They will need to know to go to these Enquiry notes rather than Submission notes otherwise they are throwing 422 errors.  Thanks.

**Comment by sanjaytkbabu** [comment]
> We will also need to update the link for BF notes on the "My bring forward notifications" and the "BF Calendar" view. They will need to know to go to these Enquiry notes rather than Submission notes otherwise they are throwing 422 errors. Thanks.

Fixed it!

File: app/src/services/enquiry.ts [hunk 1]
```diff
@@ -80,6 +80,26 @@ const service = {
...
+          activity_id: { in: activityIds }
+        }
+      });
+
+      return result.map((x) => enquiry.fromPrismaModel(x));
+    } catch (e: unknown) {
+      throw e;
+    }
+  },
+
```
Comment (kyle1morel) [inline app/src/services/enquiry.ts:102]: Consider turning this into a generic `searchEnquiries`. Reference the search function in the submission service.

Comment (sanjaytkbabu) [reply app/src/services/enquiry.ts]: done!

**Review by wilwong89 — APPROVED** [review]

**Review by kyle1morel — APPROVED** [review]

**Pull request merged**